test/common/storage/StorageBackends.spec.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env node, mocha */
+describe('Storage Features Test', function () {
```

Comment:
These are pretty close to integration tests so they might be better suited to move in that testing suite. (They test the integration of our storage adapters with the actual storage backends.) Currently, the tests are organized by the type of test (unit, integration, etc) but "common" reflects the deepforge location.

This will also let us remove the script entry for `test-common`

Reply:
This file has been moved to `test/integration`